File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteInsightGroupExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteInsightGroupExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            new InsightsScoresBetaApi(config).DeleteInsightGroup(
                insightGroupKey: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling InsightsScoresBetaApi#DeleteInsightGroup: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/DeleteInsightGroupExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.InsightsScoresBetaApi(configuration).deleteInsightGroup(
  "insightGroupKey_string", // insightGroupKey
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling InsightsScoresBetaApi#deleteInsightGroup:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
